client/state/posts/selectors.js

```diff
@@ -315,6 +316,23 @@ export function getPostEdits( state, siteId, postId ) {
 }
 
 /**
+ * Returns an object of normalized edited post attributes for the site ID post ID pairing.
+ *
+ * @param  {Object} state  Global state tree
+ * @param  {Number} siteId Site ID
+ * @param  {Number} postId Post ID
+ * @return {Object}        Normalized Post revisions
+ */
+export function getNormalizedPostEdits( state, siteId, postId ) {
```

Comment:
Do we need to distinguish this from `getPostEdits`, or should we build normalization in there?

As background, we distinguish `getPost` from `getNormalizedPost` because edits need the raw post (from `getPost`) whereas display contexts want the normalized post (`getNormalizedPost`).